docs/src/upgrade/release-3.4.x.asciidoc

```diff
@@ -177,6 +177,17 @@ when dealing with that event. To make this easier, the event now raises with a `
 
 link:https://issues.apache.org/jira/browse/TINKERPOP-1831[TINKERPOP-1831]
 
+==== Gremlin.Net: Configurable Max and Min ConnectionPool Sizes
+
+Gremlin.Net's `ConnectionPool` now has a minimum and a maximum size. These sizes are configurable through added
```

Comment:
Minor grammatical fix by adding the phrase "the newly" -  "These sizes are configurable through the newly added `ConnectionPoolSettings`".